content/docs/account-holders/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Account Holders"
+linkTitle: "Account Holders"
+weight: 20
+description: >
+    Start interacting with the Autonity network, create an account and start it to issue simple transactions.
```

Comment:
Don't know if this was the intended meaning, but imo makes more sense
```suggestion
    Start interacting with the Autonity network, create an account and fund it to issue simple transactions.
```

Reply:
Agree.